### ISSUE TITLE
RMSE Performance of NNMF Model Reporting on your SVINNMF Model. 

### ISSUE THREAD

**Issue by JoonyoungYi** [jstol/neural-net-matrix-factorization#3]
Hi. First of all, I am very impressed of your implementation of NNMF model.
You said, in the SVINNMF paper, that the RMSE result of NNMF model on ML-100K is 0.9380. Right?
But, in my experiments, I achieve the RMSE 0.906 on that data. This result is consistent with the original NNMF paper's claim.
I think that you should you RMSPropOptimizer with learning rate 1e-3 and full batch including bias on MLP(I think that you're aware of this issue seeing the `fix` branch on this repository). I think these will leads you to enhance performance of your NNMF implementation.
I already uploaded my implementation. If you interested in this issue, please check my commit d63e7b134f24fff59baa90fb0cbf96befe8d479e in [my NNMF repository](https://github.com/JoonyoungYi/NNMF-tensorflow) and feel free to contact me. Thanks 👍 

p.s. I really appreciated of your efforts on implementing NNMF model.

**Comment by JoonyoungYi** [comment]
+ I used lambda 50. Because my validation case said that NNMF model on ML-100K data has best RMSE performance with lambda 47.xxxx.
